scss/tokens/_semantic.scss

```diff
@@ -13,3 +13,58 @@ $ouds-opacities: (
   "opaque":      $ouds-opacity-900
 ) !default;
 // scss-docs-end ouds-opacities-variables-maps
+
+// Elevation
+
+// scss-docs-start ouds-sem-shadow
+$ouds-elevation-x-none:                      $ouds-elevation-x-0 !default;
+$ouds-elevation-x-raised:                    $ouds-elevation-x-0 !default;
+$ouds-elevation-x-drag:                      $ouds-elevation-x-0 !default;
+$ouds-elevation-x-overlay-default:           $ouds-elevation-x-0 !default;
+$ouds-elevation-x-overlay-emphasized:        $ouds-elevation-x-0 !default;
+$ouds-elevation-x-sticky-default:            $ouds-elevation-x-0 !default;
+$ouds-elevation-x-emphasized:                $ouds-elevation-x-0 !default;
+$ouds-elevation-x-navigation-scrolled:       $ouds-elevation-x-0 !default;
```

Comment:
`$ouds-elevation-x-navigation-scrolled` doesn't exist and should be `$ouds-elevation-x-sticky-navigation-scrolled`

Reply:
Fixed by https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2710/commits/70df90752fb68034339d3de8f134533f2d0c14a4